utt/libutt/src/api/globalParams.cpp

```diff
@@ -41,10 +38,19 @@ void GlobalParams::init() {
 #endif
 
   RangeProof::Params::initializeOmegas();
-  params.reset(new libutt::Params());
+}
+
+GlobalParams GlobalParams::create(void* initData) {
```

Comment:
Why do we hide the type of the initial data? How de we guard from errors?

Reply:
Because we use forward declarations, I prefer to avoid from letting the user to pass the internal libutt types